guide/english/certifications/javascript-algorithms-and-data-structures/basic-javascript/multiply-two-decimals-with-javascript/index.md

```diff
@@ -8,3 +8,34 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
 <a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
 
 <!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+JavaScript uses the * symbol for multiplication. Multiplying decimals is the same as multiplying whole numbers except you are working with decimal numbers now so you must include the decimal point and the number that follows the decimal in your equation.
```

Comment:
I would suggest that you use the standard terminology of "floats" and "integers" instead of using terms like decimal numbers and whole numbers. Also, please make it clear that JavaScript has only "Number" type and no specific type for "integers'